packages/debug-ui/package.json

```diff
@@ -35,6 +35,8 @@
   },
   "dependencies": {
     "@joincivil/core": "^2.0.0",
+    "bignumber.js": "5.0.0",
+    "fs": "^0.0.1-security",
```

Comment:
like node's fs? don't add it to dependencies. Add `@types/node` to devdependencies and that's it.

Reply:
It's not gonna work in web anyways